Modix/Services/Cat/CatService.cs

```diff
@@ -16,6 +16,11 @@ public struct URL
             public string file { get; set; }
         }
 
+        /// <summary>
+        /// Gets the cat GIF.
+        /// </summary>
+        /// <param name="token">The token.</param>
+        /// <returns>Return the URL to the cat</returns>
```

Comment:
I smell stock StyleCop comments -- my opinion on these is to either update them to be more useful than the method name and parameters suggest, or remove them.